File: Q04.hs
main :: IO ()
main = do
    putStrLn "Qual a sua idade:"
    input <- getLine
    let idade = read input :: Int

    if idade >= 0 && idade <18 
    then do putStrLn "Você é menor de idade."
    else if idade >= 18 
    then do putStrLn "Você é maior de idade."
    else do putStrLn "Inválido."

File: Q04.java
import java.util.Scanner;

class quartaClasse {
    public static void main(String[] args) {
        Scanner myObj = new Scanner(System.in);
        System.out.println("Qual a sua idade?");
        int idade = myObj.nextInt();

        if (idade >= 0 && idade <18){
            System.out.println("Menor de idade.");
        } else if (idade >=18) {
            System.out.println("Maior de idade.");
        }
    }
}
